apps/daffio/src/app/core/router/effects.ts

```diff
@@ -0,0 +1,10 @@
+import { inject } from '@angular/core';
+import { createEffect } from '@ngrx/effects';
+
+import { DaffRouterActivatedRoute } from '@daffodil/router';
+
+// TODO: should this be moved to `@daffodil/router/state`?
+export const daffioRouterActivatedRoute = createEffect(
```

Comment:
What's the difference between this and a regular observable of the route? That it can be loaded via EffectModule?

Reply:
yeah it felt appropriate to put it in an effect but there's no real functional benefit over just subbing in the `AppModule`

Reply:
Is there a way that we can make something like "pseudoeffects" that automatically sub in the AppModule? I'd like to keep a safe distance ( as best we can) between @ngrx and anything router related.